### DISCUSSION TITLE
visual interface for easy deployment and testing

### DISCUSSION THREAD

**Discussion by seoeaa** [safevideo/autollm#111] [Feature Requests]
I would like to know what product you want to create, there is a road map.

In general, it would be great if it had a visual interface, simple and easy,
Inside is a button to create an application.

And when you create an application, you select a supplier, embedding model, storage, etc. and a button to enable the API.

You can add a chat widget to test it right away.

And even better is the functionality of embedding a widget on a website and customizing its design.

**Comment by fcakyon** [comment]
Yes @seoeaa, this is in our road map! We plan to implement the ui's and widgets with [gradio](https://www.gradio.app/) 👍
